src/Microsoft.ML.Experimental/DataLoadSave/Database/DatabaseLoader.cs

```diff
@@ -287,6 +306,9 @@ public Range(int min, int max)
             [Argument(ArgumentType.AtMostOnce, HelpText = "Last index in the range")]
             public int Max;
 
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Name of the column")]
+            public string Name;
```

Comment:
Maybe `Name` is OK, since the XML class doc for Range says it represents input columns. Maybe if we just update the help text here to be `Name of the input column`?

---
In reply to: [332550925](https://github.com/dotnet/machinelearning/pull/4308#discussion_r332550925) [](ancestors = 332550925)

Reply:
Changed.